pkg/config/config.go

```diff
@@ -45,7 +46,7 @@ type Config struct {
 	httpsProxy    string
 	noProxy       string
 	skipTLSVerify string
-	populated     bool
+	SkipCertCheck bool
```

Comment:
Not a big fan of this approach, I would just make it configurable where the certs get read/written to and then have a unit test that uses a temp dir or a Reader/Writer